### PR TITLE
Updates after testing

### PR DESCRIPTION
**Pull request by jmudge**

### CODE REVIEW

File: Overviews/autho4api_overview.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Authorisation Framework for Network APIs Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/autho4api_overview.html]: No need for this javascript unless you wanted the same functionality as the specifications page(s).

i.e. those that have an id="releases-table"

File: Overviews/autho4api_overview.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/autho4api_overview.html]: Same as the above comment for the javascript file.

File: Overviews/browsing_overview.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Browsing Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/browsing_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/browsing_overview.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/browsing_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/dm_overview.html [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Device Management Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/dm_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/dm_overview.html [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/dm_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/lightweightm2m_overview.html [hunk 7]
```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - LightweightM2M V1.0 Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/lightweightm2m_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/lightweightm2m_overview.html [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/lightweightm2m_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/presencesimple_overview.html [hunk 9]
```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Presence SIMPLE Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/presencesimple_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/presencesimple_overview.html [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/presencesimple_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/sec_cf_overview.html [hunk 11]
```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html>
+<head>
+
+    <title>Open Mobile Alliance - Application Layer Security Common Functions Overview</title>
+    <meta charset="utf-8">
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script src="Scripts/specifications.js"></script>
```
Comment (nathan-omaorg) [inline Overviews/sec_cf_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/sec_cf_overview.html [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" href="CSS/Normalize.css" />
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+    <!-- Optional theme -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css">
+
+    <link rel="stylesheet" href="CSS/Site.css">
+    <link rel="stylesheet" href="CSS/specifications.css">
```
Comment (nathan-omaorg) [inline Overviews/sec_cf_overview.html]: Same as above comments regarding not needed unless having a releases-table

File: Overviews/sec_cf_overview.html (hunk 11 above)
Comment (jmudge) [inline Overviews/sec_cf_overview.html]: Thanks for your feedback Nathan. I don't know if these overview pages need a table as they have not been completed yet. As suggested I have removed the highlighted js and css links.

File: Overviews/sec_cf_overview.html (hunk 12 above)
Comment (jmudge) [inline Overviews/sec_cf_overview.html]: Thanks for your feedback Nathan. I don't know if these overview pages need a table as they have not been completed yet. As suggested I have removed the highlighted js and css links.

**Pull request merged**